src/vs/platform/terminal/common/xterm/shellIntegrationAddon.ts

```diff
@@ -128,6 +128,7 @@ export class ShellIntegrationAddon extends Disposable implements IShellIntegrati
 	private _activationTimeout: any;
 
 	constructor(
+		_disableTelemetry: boolean | undefined,
```

Comment:
No longer used

Reply:
doesn't work when I remove it

Reply:
bc is needed for the renderer

Reply:
The variable is never used though, we use `!this._telemetryService` to disable now

Reply:
yeah it should've been used. pushed